lms/js_config_types.py

```diff
@@ -63,6 +63,14 @@ class DashboardRoutes(TypedDict):
     course: str
     course_assignment_stats: str
 
+    organization_courses: str
+
+
+class User(TypedDict):
+    is_staff: bool
```

Comment:
We can use is_staff to provide an additional UI hint that the current use is staff.

This is not as important if we are going to display display_name, it would be an email address for staff members.